packages/playground/Gemfile.lock

```diff
@@ -18,8 +18,8 @@ GEM
       http_parser.rb (~> 0.6.0)
     eventmachine (1.2.7)
     eventmachine (1.2.7-x64-mingw32)
-    ffi (1.11.1)
-    ffi (1.11.1-x64-mingw32)
+    ffi (1.15.5)
```

Comment:
I don't think that this should be included in the PR.

Reply:
we decided to leave it because its needed for windows to run the playground

Reply:
I wonder why is this generated .lock file by Ruby not git ignored, to me it seems the same as to leave the package.lock tracked